.github/workflows/node.js.yml

```diff
@@ -0,0 +1,39 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install ksc
+        run: |
+          sudo apt-get update
+          curl -fsSL -O https://github.com/kaitai-io/kaitai_struct_compiler/releases/download/0.9/kaitai-struct-compiler_0.9_all.deb
+          sha256sum --check --warn .github/checksums
+          sudo apt-get install ./kaitai-struct-compiler_0.9_all.deb
+          sudo apt-get install --no-install-recommends -y \
+            kaitai-struct-compiler default-jre-headless
```

Comment:
Trying to install `kaitai-struct-compiler` for the second time (when it's been already installed from GitHub Releases) won't do anything, it's just confusing. It prints this (https://github.com/kaitai-io/kaitai_compress/runs/3225416347?check_suite_focus=true#step:4:84):

```
Reading package lists...
Building dependency tree...
Reading state information...
kaitai-struct-compiler is already the newest version (0.9). 
```

I see that it's my fault that I [left it there in KSF](https://github.com/kaitai-io/kaitai_struct_formats/blame/fa70a3e1baf90517db1145d3c552430bc8d8ad68/.github/workflows/main.yml#L30), sorry for that.

Reply:
Yep, you're correct, I've just copy-pasted it from there, and in reality it's not necessary. Thanks!